### ISSUE TITLE
Delete Contact Information CC - Large buttons are all missing accessibility hints

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#22296]
## What happened?
All of the large buttons for this (Remove [phone] button, Confirm and Cancel) are missing their accessibility hints.

## Specs:
- Device: Android and iOS with screen reader on

## Steps to Reproduce
- Visit phone pages and read buttons for removal with screen reader on.

## Desired behavior
Accessibility Implementation Guidelines

Remove [page name] button:
accessibilityLabel= “Remove [page name]”
accessibilityRole= "button"
accessibilityHint= "Remove [page name] from profile"

Mark "Are you sure you want to delete your [page name]?" as Header 2
Confirm button:
accessibilityLabel= "Confirm"
accessibilityRole= "button"
accessibilityHint= "Confirm you want to delete your [page name] and go back to the Personal Information page"

Cancel button:
accessibilityLabel= "Cancel"
accessibilityRole= "button"
accessibilityHint= "Cancel and continue editing"

## Acceptance Criteria
Accessibility hints are present.

## Bug Severity
- 4 - Low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/20334

**Comment by digitalken** [comment]
PASS 6/25/2021 - Sounds good on iOS and Android.  Closing.
